src/handbook/development/ops/maintenance.md

```diff
@@ -0,0 +1,76 @@
+---
+navTitle: Infrastructure Maintenance
+---
+
+Periodically, FlowFuse performs maintenance on FlowFuse Cloud resources. This page describes how we manage and execute maintenance tasks around the infrastructure.
+
+# Maintenance scope
+
+The maintenance most often involves updates to the following resources:
+* underlying cluster hosts
+* Kubernetes versions
+* databases
+* networking components
+
+ 
+# Maintenance process
+
+## Planning
+
+The maintenance process is initiated by the Operations Team. The process is planned in advance and communicated to the customers. The maintenance window is scheduled to minimize the impact on the team and customers.
+
+Within a team, a maintenance date should be agreed upon, and the team should be informed about the maintenance window.
+
+While picking the correct date and time for maintenance, the following should be considered:
+* at least two weeks' notice should be provided to customers
+* maintenance should not overlap with [scheduled release](../releases/process.md) week
```

Comment:
```suggestion
* maintenance should not overlap with [scheduled release](../releases/process.md) week
* at a time to minimise disruption to customers; outside of core business hours or the weekend
```

We should scheduled disruptive maintenance to be outside core business hours by default.